src/js/controller/controller.js

```diff
@@ -178,7 +178,7 @@ Object.assign(Controller.prototype, {
             const index = model.get('item') + 1;
             const recsAuto = (model.get('related') || {}).oncomplete === 'autoplay';
             let item = model.get('playlist')[index];
-            if ((item || recsAuto) && Features.backgroundLoading) {
```

Comment:
Assign `model.get('backgroundLoading')` as a class property so that we don't need to keep reading it from the model.